ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/StaticQueryExecutor.java

```diff
@@ -151,7 +152,7 @@ public static void validate(
 
       final WhereInfo whereInfo = extractWhereInfo(analysis, query);
 
-      final QueryId queryId = uniqueQueryId();
+      final QueryId queryId = new QueryId(QUERY_ID_PREFIX + getSourceName(analysis).name());
```

Comment:
Hummm... the issue I see here is that there can be different pull queries against the same store and they're all going to get the same id.  This will make tracking down errors more troubling.  At the moment the queryId is returned in the response, so users are able to filter the processing log to get just the errors for their invocation.  So I see this as a step backwards.

I think a better solution, and the one I was planning, is to extract the query id out of the cached mappers, filters, etc and pass it in at execution time.